source/code/source/playfab/PlayFabWinHttpPlugin.cpp

```diff
@@ -372,10 +370,15 @@ namespace PlayFab
     void PlayFabWinHttpPlugin::SetPredefinedHeaders(const CallRequestContainer& requestContainer, HINTERNET hRequest)
     {
         UNREFERENCED_PARAMETER(requestContainer);
-        WinHttpAddRequestHeaders(hRequest, L"Accept: application/json", -1, 0);
-        WinHttpAddRequestHeaders(hRequest, L"Content-Type: application/json; charset=utf-8", -1, 0);
-        WinHttpAddRequestHeaders(hRequest, (L"X-PlayFabSDK: " + std::wstring(PlayFabSettings::versionString.begin(), PlayFabSettings::versionString.end())).c_str(), -1, 0);
-        WinHttpAddRequestHeaders(hRequest, L"X-ReportErrorAsSuccess: true", -1, 0);
+        setPredefinedHeadersFailed = TryAddHeader(hRequest, L"Accept: application/json") &&
```

Comment:
>setPredefinedHeadersFailed [](start = 8, length = 26)

isn't this backwards? TryAddHeader returns TRUE on success.

when all of the TryAddHeader calls succeed you are setting `setPredefinedHeadersFailed` to TRUE which would indicate a failure, not success